ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -745,6 +754,78 @@ internal void OnNewLicenseList() {
 		return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 	}
 
+	private async Task<string?> ResponseAdvancedLootByAssetRarity(EAccess access, string targetAppID, string targetContextID, string assetRaritiesText) {
+		if (!Enum.IsDefined(access)) {
+			throw new InvalidEnumArgumentException(nameof(access), (int) access, typeof(EAccess));
+		}
+
+		ArgumentException.ThrowIfNullOrEmpty(targetAppID);
+		ArgumentException.ThrowIfNullOrEmpty(targetContextID);
+		ArgumentException.ThrowIfNullOrEmpty(assetRaritiesText);
+
+		if (access < EAccess.Master) {
+			return null;
+		}
+
+		if (!Bot.IsConnectedAndLoggedOn) {
+			return FormatBotResponse(Strings.BotNotConnected);
+		}
+
+		if (!uint.TryParse(targetAppID, out uint appID) || (appID == 0)) {
+			return FormatBotResponse(Strings.FormatErrorIsInvalid(nameof(appID)));
+		}
+
+		if (!ulong.TryParse(targetContextID, out ulong contextID) || (contextID == 0)) {
+			return FormatBotResponse(Strings.FormatErrorIsInvalid(nameof(contextID)));
+		}
+
+		string[] assetRaritiesArgs = assetRaritiesText.Split(SharedInfo.ListElementSeparators, EAssetRarities + 1, StringSplitOptions.RemoveEmptyEntries);
+
+		switch (assetRaritiesArgs.Length) {
+			case 0:
+				return FormatBotResponse(Strings.FormatErrorIsEmpty(nameof(assetRaritiesArgs)));
+			case > EAssetRarities:
```

Comment:
After constant disappears, this whole switch is pending removal. You can't have `0` entries with non-empty `assetRaritiesText`, and the second condition will no longer be possible to check. Not like we need it anyway.